src/app/_services/form-constants.service.ts

```diff
@@ -257,11 +257,11 @@ export class FormConstantsService {
     { code: 'ZMB', name: 'Zambia' },
     { code: 'ZWE', name: 'Zimbabwe' }
   ];
-  private categories = ['Non-Profit'];
+  private categories = [{name: 'Non-Profit', value: 'N'}];
 
   constructor() { }
 
-  getCategories(): string[] {
+  getCategories(): any[] {
```

Comment:
an easy definition could be 
```
getCategories(): {[key: string]: string} {
```

[sic]

or you could use a `Map<string, string>` type possibly, although i haven't had much experience with that.